src/bsoncxx/document/element.hpp

```diff
@@ -104,44 +90,20 @@ class BSONCXX_API element {
     ///
     const std::uint8_t* raw() const;
 
-    ///
```

Comment:
This and the next two functions were unsafe and unused. Too easy to tweak an element into a bad state. Element is supposed to be an immutable view anyway. What good would these do you?